api/overview/azure/virtual-network.md

```diff
@@ -26,14 +28,57 @@ Install the [NuGet package](https://www.nuget.org/packages/Microsoft.Azure.Manag
 Install-Package Microsoft.Azure.Management.Network.Fluent
 ```
 
+#### .NET Core CLI
+
 ```bash
 dotnet add package Microsoft.Azure.Management.Network.Fluent
 ```
 
-> [!div class="nextstepaction"]
-> [Explore the management APIs](/dotnet/api/overview/azure/network/management)
+### Code Example
+This example shows how you can create a virtual network.
+
+```csharp
+/* 
+  Include these "using" directives...
+  
+  using Microsoft.Azure.Management.Network.Fluent;
+  using Microsoft.Azure.Management.Network.Fluent.Models;
+*/
+using (NetworkManagementClient client = new NetworkManagementClient(credentials))
+{
+    // Define VNet
+    VirtualNetworkInner vnet = new VirtualNetworkInner()
+    {
+        Location = "West US",
+        AddressSpace = new AddressSpace()
+        {
+            AddressPrefixes = new List<string>() { "0.0.0.0/16" }
+        },
+
+        DhcpOptions = new DhcpOptions()
+        {
+            DnsServers = new List<string>() { "1.1.1.1", "1.1.2.4" }
+        },
 
+        Subnets = new List<Subnet>()
+        {
+            new Subnet()
+            {
+                Name = subnet1Name,
+                AddressPrefix = "1.0.1.0/24",
+            },
+            new Subnet()
+            {
+                Name = subnet2Name,
+               AddressPrefix = "1.0.2.0/24",
+            }
+        }
+    };
+   await client.VirtualNetworks.CreateOrUpdateAsync("resourceGroupName", "vNetName", vnet);
+}
 
+```
+
+> [!div class="nextstepaction"]
+> [Explore the management APIs](/dotnet/api/overview/azure/network/management)
 
-[PackageManager]: https://docs.microsoft.com/nuget/tools/package-manager-console
-[DotNetCLI]: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-add-package

```

Comment:
I believe these are needed at the bottom because of a special way they link to the references earlier in the document, so should not be removed (if you look at other published examples, you'll see them). Only caveat is that you need to remove the /en-us/ portion.